SDK implementation for Radiology Insights (in public preview)

koen-mertens opened the pull request:
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

asaflevi-ms committed:
init

asaflevi-ms committed:
Merge branch 'main' into asaflevi/healthinsights-radiologyinsights-net-sdk

asaflevi-ms committed:
Radiology insights initial

rameezmicrosoft committed:
hero scenario test case

rameezmicrosoft committed:
Hero scenario test case

rameezmicrosoft committed:
Ranamed hero scenario to RadiologyInsightsClientSample and move it to the samples directory

asaflevi-ms committed:
Merge branch 'main' into asaflevi/healthinsights-radiologyinsights-net-sdk

asaflevi-ms committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

asaflevi-ms committed:
fix: token-expected: ';'

asaflevi-ms committed:
Revert "Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk"

This reverts commit f19019351e575e142440e699ffc98a9bbbc8cbc5, reversing
changes made to f93993ee6962722c00de52cedcd0ee2ad34be559.

rameezmicrosoft committed:
changes for CriticalResult sdk

koen-mertens committed:
Compliance with requirements for RadiologyInsights .NET SDK

rameezmicrosoft committed:
changed return type for async call

rameezmicrosoft committed:
changes for sdk live tests

rameezmicrosoft committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

rameezmicrosoft committed:
more inference samples

rameezmicrosoft committed:
corrected offsets

rameezmicrosoft committed:
changes for inferences

rameezmicrosoft committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

rameezmicrosoft committed:
followupcommunication inference

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.Health.Insights.RadiologyInsights](https://apiview.dev/Assemblies/Review/d43daa369e2a49c0bb24a37339ceacea?revisionId=6e83fab3395246c889906fee5d2a18f0)

rameezmicrosoft committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

koen-mertens committed:
Update README.md

koen-mertens committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

koen-mertens committed:
Update README.md

remove links that don't exist yet

rameezmicrosoft committed:
recorded tests

rameezmicrosoft committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

rameezmicrosoft committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

koen-mertens committed:
remove samples

koen-mertens committed:
clean up samples

rework samples and readme files

koen-mertens commented:
/azp run prepare-pipelines

rameezmicrosoft committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

koen-mertens committed:
add readme samples and adjust tsp-location

- change the commit id to be in sync with the latest version.
- add readme files for the 2 samples

koen-mertens committed:
Update README.md

rameezmicrosoft committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

koen-mertens committed:
Merge branch 'Azure:main' into asaflevi/healthinsights-radiologyinsights-net-sdk

rameezmicrosoft committed:
Merge branch 'asaflevi/healthinsights-radiologyinsights-net-sdk' of https://github.com/asaflevi-ms/azure-sdk-for-net into asaflevi/healthinsights-radiologyinsights-net-sdk

rameezmicrosoft committed:
generated files using TypeSpec-Project-Generate.ps1

chunyu3 reviewed (dismissed):
Recording need to be submit to `Azure SDK assets` repo. you can refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/README.md#recording

rameezmicrosoft committed:
changes after running sync, generate and export scripts

rameezmicrosoft committed:
changes for providing endpoint and key from environment

koen-mertens committed:
Merge branch 'Azure:main' into asaflevi/healthinsights-radiologyinsights-net-sdk

koen-mertens committed:
adding assets for net

koen-mertens committed:
Update CHANGELOG.md

koen-mertens committed:
Update CODEOWNERS

jsquire reviewed (dismissed):
@koen-mertens:

Please make your Azure organization membership public on your GitHub profile.  Looks like you've got the correct repository permissions, but your membership is private.

For information and assistance, please review the Azure SDK onboarding documentation  and use the Onboarding Teams channel:
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

koen-mertens commented:
> @koen-mertens:
> 
> Please make your Azure organization membership public on your GitHub profile. Looks like you've got the correct repository permissions, but your membership is private.
> 
> For information and assistance, please review the Azure SDK onboarding documentation and use the Onboarding Teams channel:
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

I did change it yesterday. Hope it is adjusted by now…

jsquire commented:
> > @koen-mertens:
> > Please make your Azure organization membership public on your GitHub profile. Looks like you've got the correct repository permissions, but your membership is private.
> > For information and assistance, please review the Azure SDK onboarding documentation and use the Onboarding Teams channel:
> > 
> > * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> > * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> I did change it yesterday. Hope it is adjusted by now…

Confirmed.  I'm seeing it as public now.   I've kicked the linter job to recheck; should pass now.

koen-mertens committed:
Update CHANGELOG.md

koen-mertens committed:
Merge branch 'Azure:main' into asaflevi/healthinsights-radiologyinsights-net-sdk

asaflevi-ms committed:
After sync with latest and regenerate the SDK

asaflevi-ms commented on .github/CODEOWNERS:
```diff
@@ -232,7 +232,7 @@
 
 # PRLabel: %Cognitive - Health Insights
-/sdk/healthinsights/                                               @asaflevi-ms
+/sdk/healthinsights/                                               @asaflevi-ms @tomsft @koen-mertens
```
I think you can add your names under radiology insights and not entire health insights SDKs.

rameezmicrosoft committed:
removed unnecessary generated and suppressed

asaflevi-ms commented:
Approved
